bind/flag.go

```diff
@@ -105,6 +105,29 @@ func HTTPProxyConfig(fs *pflag.FlagSet, cfg *forwarder.HTTPProxyConfig, lcfg *lo
 			"By default, requests to localhost are denied. ")
 }
 
+func MITMConfig(fs *pflag.FlagSet, mitm *bool, cfg *forwarder.MITMConfig) {
+	fs.BoolVar(mitm, "mitm", *mitm, ""+
+		"Enable Man-in-the-Middle (MITM) mode. "+
+		"It only works with HTTPS requests, HTTP/2 is not supported. "+
+		"MITM is enabled by default when the --mitm-cacert-file flag is set. "+
+		"If the CA certificate is not provided MITM uses a generated CA certificate. "+
```

Comment:
> If the CA certificate is not provided MITM uses a generated CA certificate

I don't see a use case for that. I think we must require a cert when MITM is enabled

Reply:
This is very useful for quick checks.
Then you can get the certificate from the API server and not bother generating it.
We already have sth very similar for running HTTPS with self-signed cert.

Reply:
Think Windows.